utils/syscall.c

```diff
@@ -583,7 +584,14 @@ static int _process_syscall_group[] = {SYS_clone,
                                        SYS_tgkill,
                                        SYS_waitid,
                                        SYS_rt_tgsigqueueinfo,
-                                       SYS_execveat};
+                                       SYS_execveat,
+                                       SYS_myst_get_process_thread_stack,
+                                       SYS_myst_get_exec_stack_option,
+                                       SYS_myst_pre_launch_hook,
```

Comment:
Consider moving this one to `myst` group as we discussed.

Reply:
Added this!